Exercise_D_E/assets/style/index.css

```diff
@@ -2,3 +2,127 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+* {
+    margin:0;
+}
+.header{
+
+    width: 100%;
+    height: 70px;
+    
+}
+
+.image1{
+    background-image: url(../images/logo.png);
+    height: 50px;
+    width: 200px;
+    
+}
+
+.main{
+    display: flex;
+    flex-direction: column;
+    height: 500px;
+    width:100%;
+
+    background-color: rgba(0,0,0, 0.5);
+    align-items: center;
+    justify-content: center;
+}
+
+.bigImage{
+    position: absolute;
+    width: 100%;
+    height: 500px;
+    background-image: url(../images/flowershop.jpg);
+    opacity: 0.5;
+    z-index: -1;
+}
+.maintitle{
+    color: white;
+}
+
+.button{
+    background-color: green;
+    height: 35px;
+    width: 130px;
+    border-radius: 50px;
+    display:flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    top: 50px;
+}
+
+.learnmore{
+    font-size: 20px;
+    color: white;
+}
+
+.footer{
```

Comment:
Hey Anson, 
What i would suggest for this part and also for better responsiveness is to change this to 
`.footer{
height:100vh;
width:100vw;
}`